station-info/README.md

```diff
@@ -0,0 +1,28 @@
+# Fetch station details with Chargetrip API
+
+This tutorial explains how to fetch station details and suggests how to parse the data.  
+To see this example live 👉 [demo](https://chargetrip.github.io/examples/station-info).
+
+### Technical stack
+
+The Chargetrip API is built around GraphQL. If you aren't familiar with GraphQL, [going over the specs](https://graphql.org/learn/) will be helpful. Don't worry, you don't need to be an expert to use this API, this getting started guide should be enough to get going.
+
+To see our Chargetrip API in action, you can go to the [Playground](https://playground.chargetrip.com/). It has a big collection of mutations/queries for you to experience the power of our API.
+
+This example is built with vanilla JS. To establish a connection with Chargetrip API, we use [urql](https://formidable.com/open-source/urql/) - lightweight GraphQL client.
+We use our Playground environment for this example. Our Playground has a station database that is populated with freely available European station data from [OCM](https://openchargemap.org/site). It means that only part of our extensive database is available. You need a registered `x-client-id` to access the full database. Importing your own database or using one of the databases Chargetrip has an integration with, is possible. For more details, please contact us.
+
+### Steps to take
+
+To get stations around some location and fetch specific station details:
```

Comment:
I would not use around some location but around a location or maybe even around a specific location.

Reply:
👍